src/api/Server.ts

```diff
@@ -126,6 +126,10 @@ export class FosscordServer extends Server {
 		app.use("/api/v9", api);
 		app.use("/api", api); // allow unversioned requests
 
+		try {
+			require("./middlewares/TestClient").default(this.app);
+			// eslint-disable-next-line no-empty
+		} catch (error) {}
```

Comment:
why? you can just use something like https://github.com/V3L0C1T13S/diproxi, or if you have the files just run a local http server